### PR TITLE
Rework some of Dropdown styles to make chat options work

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Resolves
- https://github.com/sparkletown/internal-sparkle-issues/issues/1723

Partially resolves
- https://github.com/sparkletown/internal-sparkle-issues/issues/1692

|**BEFORE** | **AFTER**|
|-------------|-----------|
|sparkle-chat-options-02|sparkle-chat-options-01|

### CODE REVIEW

File: src/components/atoms/Dropdown/Dropdown.scss [hunk 1]
```diff
@@ -37,12 +41,12 @@
 
     &__menu {
       background: var(--content--under-lighter-20pp);
-      box-shadow: 0 10px 20px opaque-black(0.25);
+      box-shadow: 0 10px 20px var(--content--under-25a);
```
Comment (alexey-petrov) [inline src/components/atoms/Dropdown/Dropdown.scss:44]: i think we can use a box-shadow function from `constants.scss` or add one there, but it's up to you

File: src/components/atoms/Dropdown/Dropdown.scss [hunk 2]
```diff
@@ -73,10 +77,22 @@
       padding: 0;
       margin: 0;
       opacity: 0;
+      visibility: visible;
+      // color: hsl(0, 0%, 20%);
```
Comment (alexey-petrov) [inline src/components/atoms/Dropdown/Dropdown.scss:81]: let's remove this

File: src/components/atoms/Dropdown/Dropdown.scss [hunk 3]
```diff
@@ -73,10 +77,22 @@
...
+      box-sizing: border-box;
     }
 
-    &__input-container &__input {
-      opacity: 0 !important;
+    &__input-container {
+      display: inline;
+
+      &__input {
+        //height: 0;
```
Comment (alexey-petrov) [inline src/components/atoms/Dropdown/Dropdown.scss:93]: remove

File: src/components/atoms/Dropdown/Dropdown.scss [hunk 4]
```diff
@@ -87,9 +103,10 @@
       display: none;
     }
 
+    // &__Select__value-container--has-value
     &__value-container {
+      //overflow: hidden; // NOTE: makes the static text move up and hide
```
Comment (alexey-petrov) [inline src/components/atoms/Dropdown/Dropdown.scss:108]: remove?

File: src/components/atoms/Dropdown/Dropdown.scss [hunk 5]
```diff
@@ -87,9 +103,10 @@
       display: none;
     }
 
+    // &__Select__value-container--has-value
```
Comment (alexey-petrov) [inline src/components/atoms/Dropdown/Dropdown.scss:106]: remove?

**Review by alexey-petrov — APPROVED** [review]
LGTM, a few minor suggestions

File: src/components/atoms/Dropdown/Dropdown.scss (hunk 1 above)
Comment (goran-peoski-work) [reply src/components/atoms/Dropdown/Dropdown.scss:44]: Good idea

File: src/components/atoms/Dropdown/Dropdown.scss (hunk 2 above)
Comment (goran-peoski-work) [reply src/components/atoms/Dropdown/Dropdown.scss:81]: I was trying to just copy the defaults from what the component generated in their custom classes. Still wary of removing them w/o first checking how it works as it's "should be".

File: src/components/atoms/Dropdown/Dropdown.scss (hunk 4 above)
Comment (goran-peoski-work) [reply src/components/atoms/Dropdown/Dropdown.scss:108]: I was considering if the comment is enough or remove it all

File: src/components/atoms/Dropdown/Dropdown.scss (hunk 5 above)
Comment (goran-peoski-work) [reply src/components/atoms/Dropdown/Dropdown.scss:106]: Keep, for now. It's one of the classes generated by the component. I should add note though.

**Pull request merged**